CMakeLists.txt

```diff
@@ -3,7 +3,9 @@ project(point-cloud-utils)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_BUILD_TYPE "Release")
 
-option(GEOGRAM_WITH_HLBFGS "Non-linear solver (Yang Liu's HLBFGS)"             ON)
+if(NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm64")
```

Comment:
below the check is for `"(arm64)|(ARM64)"`. Do we want the same check here? Maybe also `(arm32)|(ARM32)`?